service/app/controllers/usermanagement/UserStatusController.java

```diff
@@ -0,0 +1,39 @@
+package controllers.usermanagement;
+
+import controllers.BaseController;
+import controllers.usermanagement.validator.UserStatusRequestValidator;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.request.Request;
+import play.libs.F.Promise;
+import play.mvc.Result;
+
+public class UserStatusController extends BaseController {
```

Comment:
Why is this PR have UserStatusController files? These are already merged to release-1.12 branch.